handlers/main.yml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
We should avoid `when` condition inside handler. It's better to handle condition in tasks

Reply:
Ok, rework handlers